htmd/simlist.py

```diff
@@ -12,6 +12,7 @@
 from htmd.molecule.molecule import Molecule
 from joblib import Parallel, delayed
 from htmd.progress.progress import ProgressBar
+import mdtraj as md
 import logging
```

Comment:
Ok, just keep in mind that the more you modify the "base" classes of HTMD the harder it becomes to accept your pull request as the risk of breaking everything else increases. If you need modifications to these better ask us to do them.